lib/src/summation/counter.dart

```diff
@@ -17,6 +18,19 @@ class Counter extends SummationBase {
   /// The output value of the counter.
   Logic get count => output('count');
 
+  /// The main clock signal.
+  @visibleForTesting
```

Comment:
This is interesting:  I think we sometimes enable access in test and I did not know there was a specific decorator

Reply:
i also did not know about this until someone else showed me, I think it was @saw235, very useful indeed!

the full list from the `meta` package is here and there are some other good interesting ones too:
https://pub.dev/documentation/meta/latest/meta/meta-library.html